irc/handlers.go

```diff
@@ -897,9 +897,7 @@ func formatBanForListing(client *Client, key string, info IPBanInfo) string {
 	if info.OperReason != "" && info.OperReason != info.Reason {
 		desc = fmt.Sprintf("%s | %s", info.Reason, info.OperReason)
 	}
-	if info.Duration != 0 {
-		desc = fmt.Sprintf("%s [%s]", desc, info.TimeLeft())
-	}
+	desc = fmt.Sprintf("%s [%s]", desc, info.TimeLeft())
```

Comment:
Can you consolidate this and the line below into a single `fmt.Sprintf` invocation, for clarity?